example/lib/src/utils.dart

```diff
@@ -0,0 +1 @@
+void log(String text) => print("[FlutterReactiveBLE] $text");
```

Comment:
nit: maybe name it FlutterReactiveBLEAPP  to distinguish between library and app>